docs/ops/condition/NonZero_3.md

```diff
@@ -54,3 +57,16 @@ The output tensor has shape `[rank(input), num_non_zero]`. For example, for the
     </output>
 </layer>
 ```
+```
+Input tensor:   [[0, 1],
```

Comment:
This is unusual ... maybe better if it was included in detailed description?

Reply:
I removed this, in return I've added an extra line to the detailed description describing the columns in the example given there.